bundles/org.openhab.binding.emotiva/src/main/java/org/openhab/binding/emotiva/internal/EmotivaProcessorHandler.java

```diff
@@ -189,13 +188,14 @@ private synchronized void connect() {
 
             if (udpSenderActive) {
                 updateStatus(ThingStatus.ONLINE);
+                long nowEpochSecond = ZonedDateTime.now(ZoneId.systemDefault()).toEpochSecond();
```

Comment:
```suggestion
                long nowEpochSecond = Instant.EPOCH;
```

Reply:
`Instant.EPOCH` represents 1970-01-01T00:00:00Z as an `Instant`, so this is not a `long`. Second, the code above converts `now` to epoch seconds. Your suggestion would be 0 with the proper type. 😉

I have a question also though: In line 192:
```java
updateChannelState(LAST_SEEN_STATE_NAME, new QuantityType<>(nowEpochSecond, Units.SECOND));
```

Why?

This channel:

https://github.com/openhab/openhab-addons/blob/3cd5879209f40f0af8991d11b21ab748d31978f2/bundles/org.openhab.binding.emotiva/src/main/java/org/openhab/binding/emotiva/internal/EmotivaBindingConstants.java#L56

seems to not exist. And what good would a channel be containing the number of seconds since epoch?

Reply:
I checked that it no longer tries to update a non-existing channel, but didn't check the changes in detail. I'll leave that to @lsiepel, but you can consider my comment/question fully resolved. 🙂

Reply:
The last `LAST_SEEN_STATE_NAME` roque channel is still there. BindingConstants.java:56

Reply:
I still need LAST_SEEN_STATE_NAME to match the incoming keepAlive Emotiva subscription. As indicated in the value, this is not mapped to a real OH channel. I simplified it a bit, but still need the constant. This reuses the whole mapping logic for EmotivaSubscriptionTags, but I made it a bit more explicit now.